doc/openapi-user-guide.md

```diff
@@ -158,11 +161,13 @@ spec:
 | jwtClaimWithClientID     | no           | JSON Web Token (JWT) Claim with ClientID that contains the clientID. Defaults to 'azp'.                                                                                                                                                                                                                                                                                                                                                               |
 | jwtClaimWithClientIDType | no           | JwtClaimWithClientIDType sets to process the ClientID Token Claim value as a string or as a liquid template. Valid values: plain, liquid. Defaults to 'plain'                                                                                                                                                                                                                                                                                         |
 | authenticationFlow       | no           | flows object. When the sec scheme is oauth2, the flows are provided by the OpenAPI doc. However, for openIdConnect security scheme, the OpenAPI doc does not provide the flows. In that case, the OpenAPI CR can provide those. There are 4 flows parameters (for OIDC only): `standardFlowEnabled`, `implicitFlowEnabled`, `serviceAccountsEnabled`, `directAccessGrantsEnabled`. See [3scale product reference](product-reference.md) for more info |
+| gatewayResponse          | no | Specifies custom gateway response on errors. See `GatewayResponseSpec` in [3scale product reference ](product-reference.md) for more info |
 
 - **One of IssuerEndpointRef or IssuerEndpoint must be defined in OIDC Spec** (both fields can be defined, see next note).
 - **If issuerEndpoint plain value is defined in CR - it will be used as precedence over issuerEndpointRef secret**.
 - The format of issuerEndpoint is determined on your OpenID Provider setup;
   see in 3scale portal - `Product/Integration/Settings/AUTHENTICATION SETTINGS/OpenID Connect Issuer`.  
+- **There is no need to define the OIDC Security parameter in OpenApi CR; it will be populated in the product CR from the OpenApi CR if one or both of the PrivateAPISecretToken and PrivateAPIHostHeader parameters are defined in the OpenAPI CR**. See OpenAPISpec in [openapi reference](openapi-reference.md), OODC specification in [product-reference.md](product-reference.md).
```

Comment:
@valerymo if for now we want to make this a documentation update only (IMO it requires code changes as well) I would try making it clear that the hostHeader and secretToken should only be set at openAPI cr .spec level instead of at the openapi.spec.OIDC level since the OIDC value is ignored.

Reply:
updated. Thank you for comments @MStokluska